wp-parsely.php

```diff
@@ -1631,12 +1631,11 @@ private function get_settings_url() {
 	 * @param int    $post_id id of the post you want to get the url for. Optional.
 	 * @return string|void
 	 */
-	private function get_current_url( $post = 'nonpost', $post_id = 0 ) {
-		$options = $this->get_options();
-		$scheme  = ( $options['force_https_canonicals'] ? 'https://' : 'http://' );
+	public function get_current_url( $post = 'nonpost', $post_id = 0 ) {
+		$url = '';
```

Comment:
What does this assignment add?

Reply:
I do not have any strong opinion here. I just think it makes reading easier with a declaration somewhere. However, PHPStorm is also marking it's not necessary, then I am removing it.